.github/workflows/docker-image.yml

```diff
@@ -7,6 +7,14 @@ on:
         description: 'Push to Docker Hub'
         required: true
         default: 'false'
+      tags:
+        description: 'Custom tags to use for the push'
+        required: false
+        default: ''
+      ref:
+        description: 'Custom ref to check out'
+        required: false
+        default: ''
```

Comment:
I would not do that one, I would require me to first cherry pick all I need and then run top of a branch.

Reply:
OK, reverting the customisable ref.